store/tikv/region_cache.go

```diff
@@ -201,12 +226,25 @@ type RegionCache struct {
 	}
 	notifyCheckCh chan struct{}
 	closeCh       chan struct{}
+
+	labelKeys   []string
+	labelValues []string
 }
 
 // NewRegionCache creates a RegionCache.
 func NewRegionCache(pdClient pd.Client) *RegionCache {
+	var labelKeys = make([]string, 0)
+	var labelValues = make([]string, 0)
+	for _, kv := range strings.Split(config.GetGlobalConfig().Labels, ",") {
+		kvPair := strings.Split(kv, "=")
+		labelKeys = append(labelKeys, strings.Trim(kvPair[0], " \t"))
+		labelValues = append(labelValues, strings.Trim(kvPair[1], " \t"))
```

Comment:
this will panic if `config.GetGlobalConfig().Labels == ""`

Reply:
In fixing.